src/tools/kdb/editor.cpp

```diff
@@ -36,6 +32,39 @@ EditorCommand::~EditorCommand ()
 {
 }
 
+
+static kdb::Key prependNamespace (kdb::Key const & root, std::string const & ns)
```

Comment:
I think I already saw this somewhere. Maybe we can move this to libtools?

Reply:
> Maybe we can move this to libtools?

Something like that would be good I think. libtools = a new file in src/libs/tools/src ?

Reply:
Yes, and the header file could be in ~libtools/include/helper